src/main/java/net/dv8tion/jda/api/events/emote/EmoteAddedEvent.java

```diff
@@ -23,6 +23,13 @@
 
 /**
  * Indicates that a new {@link net.dv8tion.jda.api.entities.Emote Emote} was added to a {@link net.dv8tion.jda.api.entities.Guild Guild}.
+ *
+ * <h2>Requirements</h2>
+ *
+ * <p>This event requires the {@link net.dv8tion.jda.api.utils.cache.CacheFlag#EMOTE EMOTE} CacheFlag to be enabled, which requires
+ * the {@link net.dv8tion.jda.api.requests.GatewayIntent#GUILD_EMOJIS GUILD_EMOJIS} intent
```

Comment:
Use a period to end the sentence, same with the other events

Reply:
I used IntelliJ regex replace to change this. I highly doubt I missed any